content/python/concepts/sets/terms/difference-update/difference-update.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: '.difference_update()'
+Description: 'Modify a given set of objects to be unique when compared to other sets.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Collections'
+  - 'Functions'
+  - 'Methods'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.difference_unique()`** method modifies a given set of objects to be unique when compared to another set or sets. 
+
+Things to note about this method:
+- the original set is modified by discarding any of the content that appears in the sets it is compared with
+- the return value is `None`
+- as no new set is created, this method is more resource-efficient than using the [.difference()](https://www.codecademy.com/resources/docs/python/sets/difference) method, which returns a new set
+
+## Syntax
+
+The following syntaxes can be used:
```

Comment:
```suggestion
```

Reply:
I don't see any difference between the suggestion and my original, please could you elaborate? Thanks

Reply:
This suggestion means that the line "The following syntaxes can be used" can be removed.

Reply:
Ah, got it, thankyou!